kafka-ui-api/src/main/java/com/provectus/kafka/ui/exception/SchemaBaseException.java

```diff
@@ -0,0 +1,7 @@
+package com.provectus.kafka.ui.exception;
+
+public abstract class SchemaBaseException extends CustomBaseException {
```

Comment:
lets remove this one and extend SchemaFailedToDeleteException from CustomBaseException. I don't think we need another hierarchy level for this